README.md

```diff
@@ -48,9 +48,11 @@ use { "chrisgrieser/nvim-chainsaw" }
 - Ruby
 - Rust
 - CSS (and supersets)
+- Go
 
 Not every language supports every type of log statement. For details on what is
 supported, see [log-statements-data.lua](./lua/chainsaw/log-statements-data.lua).
+For languages like go, packages fmt and time are to be imported manually by the plugin user.
```

Comment:
Fmt is a builtin package?

There aren't any print statements in go without importing a package?

Reply:
yeah that is how it is

Reply:
We probably have to handle all the import scenarios properly if we are going to do it like checking if import statement is present and if present if "fmt" is present and if not adding "fmt". Same applies for "time" package as well.

Reply:
I think this will be the same when I raise PR for java as well since you have inbuilt method for printing but for timelog, there is no other way than to import LocalDateTime class from utils.

Reply:
Hmm. I think if these are standard packages the user can be expected to know about, and they are built-in, then it's tolerable.